persistence/sql/outbox_modes_sqlpersistence_[6,).partial.md

```diff
@@ -36,4 +36,10 @@ In this mode the SQL persistence creates a `TransactionScope` that wraps the who
 
 In addition to that connection managed by NServiceBus, users can open their own database connections in the message handlers. If the underlying database technology supports distributed transactions managed by Microsoft Distributed Transaction Coordinator -- MS DTC (e.g. SQL Server, Oracle or PostgreSQL), the transaction gets escalated to a distributed transaction.
 
-The `TransactionScope` mode is most useful in legacy scenarios e.g. when migrating from MSMQ transport to a messaging infrastructure that does not support MS DTC. In order to maintain consistency the outbox has to be used in place of distributed transport-database transactions. If the legacy database cannot be modified to add the outbox table, the only option is to place the outbox table in a separate database and use distributed transactions between the databases.
\ No newline at end of file
+The `TransactionScope` mode is most useful in legacy scenarios e.g. when migrating from MSMQ transport to a messaging infrastructure that does not support MS DTC. In order to maintain consistency the outbox has to be used in place of distributed transport-database transactions. If the legacy database cannot be modified to add the outbox table, the only option is to place the outbox table in a separate database and use distributed transactions between the databases.
+
+When necessary, the transaction isolation level can also be adjusted. This should be only done if the business logic that is executed by the handler within the outbox transaction requires higher isolation level to guarantee correctness (e.g. `RepeatableRead` or `Serializable`). Adjusting the isolation level is also possible when not using `TransactionScope` mode:
```

Comment:
Does this change the isolation level of the whole outbox transaction? What is the default?